src/change_stream.ts

```diff
@@ -726,6 +726,24 @@ export class ChangeStream<
     }, callback);
   }
 
+  async *[Symbol.asyncIterator](): AsyncGenerator<TChange, void, never> {
```

Comment:
```suggestion
  async *[Symbol.asyncIterator](): AsyncGenerator<TChange, void, void> {
```

I thought typescript warned against us using `never` here, can we add a type test to `test/types/change_stream.test-d.ts` that just uses the for-await syntax